src/jit/lowerxarch.cpp

```diff
@@ -1887,6 +1885,28 @@ void Lowering::TreeNodeInfoInitPutArgStk(GenTree* tree)
     TreeNodeInfo* info = &(tree->gtLsraInfo);
     LinearScan*   l    = m_lsra;
 
+#ifdef _TARGET_X86_
+    if (tree->gtOp.gtOp1->gtOper == GT_FIELD_LIST)
+    {
+        GenTreeFieldList* fieldListPtr   = tree->gtOp.gtOp1->AsFieldList();
+        for (; fieldListPtr; fieldListPtr = fieldListPtr->Rest())
```

Comment:
Be my guest :-)

---
In reply to: [79511137](https://github.com/dotnet/coreclr/pull/7252#discussion_r79511137) [](ancestors = 79511137)